packages/core/android/src/newarch/src/DdLogs.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
```

Comment:
file location doesn't respect the package name. It should be placed in the `com/datadog/reactnative` folder, same for other files.